packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -548,10 +573,21 @@ export async function getPageStaticInfo(params: {
       nextConfig
     )
 
+    if (
+      pageType === 'app' &&
+      rscInfo.hasUseClientDirective &&
+      generateStaticParams
+    ) {
```

Comment:
@ijjk Do I need to check for `output: export` here too? Seems like it also fails for normal output mode so I think it should always throw until its implemented.

Reply:
I think always throwing until implemented sounds good